js/data-analyst-agent-v2/src/schema.ts

```diff
@@ -0,0 +1,41 @@
+import { ObjectSchema } from '@microsoft/teams.ai';
+
+export const executeSqlSchema: ObjectSchema = {
+    type: 'object',
+    properties: {
+        query: {
+            type: 'string',
+            description: 'SQL query to execute'
+        }
+    },
+    required: ['query']
+};
+
+export const chartCreationSchema: ObjectSchema = {
```

Comment:
Add a comment explaining that this is a sub-schema of the entire adaptive card schema peared down to what you want the result to be like